main.go

```diff
@@ -243,6 +243,22 @@ func (f *UserHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	bookmarkings := []model.Song{}
```

Comment:
```suggestion
	var bookmarkings []model.Song
```

https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices